### PR TITLE
Creative Agency

### PR DESCRIPTION
**Pull request by LeonidShatilo**
**Requirements (max result – 5 points):**

- cross-browser layout (Chrome, Firefox, Microsoft Edge, IE11 – latest versions)
- markup should not be pixel perfect, but it must be most similar to PSD (font-size, colors, margins, etc.)
- elements, that can be hovered, must-have styles for that
- markup must be valid
- use semantic tags (section, header, footer, etc.)

**Optional (non-obligatory requirements, that increase your score) (max result – 5 points):**

- responsive (2 points) (768px, 320px – breakpoints)
- animations (1 point)
- naming convention (e.g. BEM) (2 points)

### CODE REVIEW

**Review by broshkabro — COMMENTED** [review]
**Requirements**
- [ ] cross-browser layout (chrome, ie11, edge, firefox - latest versions):
    - there are the following issues in IE 11
image
image

- [x] markup should not be pixel perfect, but it must be most similar to PSD (font-size, colours, margins, etc. Use free fonts and icons):
    -  [**FIXED**] the colour of the dashes doesn't correspond to the colour in the mock-up.
image
- [x] elements that can be hovered must-have styles for it
- [x] markup must be valid, check here:
validator
- [x] use semantic tags (section, header, footer, etc.) 

**Optional Requirements**
- [x] responsive (768, 320 breakpoints)
- [x] animations 
- [x] naming convention

**Extra**
- [x] webpack config
- [x] scss

**Comment by broshkabro** [comment]
Just recommendations:
 - pay attention to the img formats and the difference between them (PNG, JPEG/JPG)
 - pay attention to the fonts formats
 - content overflow

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,546 @@
...
+                <li class="hamburger-menu__item">
+                  <a class="hamburger-menu__item-link" href="#about">about</a>
+                </li>
+                <li class="hamburger-menu__item">
+                  <a class="hamburger-menu__item-link" href="#services"
+                    >services</a
+                  >
+                </li>
+                <li class="hamburger-menu__item">
+                  <a class="hamburger-menu__item-link" href="#portfolio"
```
[the remark this answers is not in the document]
Comment (LeonidShatilo) [reply src/index.html]: I use Prettier Formatter, but okay, I fixed the formatting.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,546 @@
...
+                    >portfolio</a
+                  >
+                </li>
+                <li class="hamburger-menu__item">
+                  <a class="hamburger-menu__item-link" href="#blog">blog</a>
+                </li>
+                <li class="hamburger-menu__item">
+                  <a class="hamburger-menu__item-link" href="#contact"
```
[the remark this answers is not in the document]
Comment (LeonidShatilo) [reply src/index.html]: I use Prettier Formatter, but okay, I fixed the formatting.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,546 @@
...
+                      stroke-linejoin="round"
+                      stroke-width="32"
+                    />
+                  </svg>
+                </div>
+                <h3 class="services__name">design & development</h3>
+                <span class="services__item">information architecture</span>
+                <span class="services__item">interface design</span>
+                <span class="services__item">product design</span>
+                <span class="services__item">integrated ad companies</span>
```
[the remark this answers is not in the document]
Comment (LeonidShatilo) [reply src/index.html]: Done!

File: src/styles/components/_title.scss [hunk 4]
```diff
@@ -0,0 +1,78 @@
+.title {
+  position: relative;
+  display: block;
+  margin: 22px 0 17px 0;
```
[the remark this answers is not in the document]
Comment (LeonidShatilo) [reply src/styles/components/_title.scss]: Changed it.

**Comment by LeonidShatilo** [comment]
Fixed the layout for IE11. Replaced images with JPG and optimized them. Also added other font formats.

**Review by broshkabro — APPROVED** [review]
Splendid!

**Pull request merged**